src/main/java/com/epam/university/java/core/task004/Dictionary.java

```diff
@@ -0,0 +1,36 @@
+package com.epam.university.java.core.task004;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by ilya on 04.09.17.
+ */
+public class Dictionary {
+    private final static Map<String,Integer> DICTIONARY= new HashMap<>();
```

Comment:
It's better not to hardcode this =)

Reply:
Not to do dictionary or not to do dictionary like this?

Reply:
Not to do dictionary or not to do dictionary like this?

Reply:
Dictionary like this. Strings here are just for example, here can be another words.